### PR TITLE
Queues: Update commands.md

### PR DESCRIPTION
**Pull request by elithrar**
This PR:

* Adds Queues to the wrangler CLI docs.
* Links to the Wrangler CLI docs from the Queues docs, identical to D1 and Hyperdrive.

### CODE REVIEW

File: content/workers/wrangler/commands.md [hunk 1]
```diff
@@ -1704,6 +1705,92 @@ This command has been deprecated as of v3 in favor of [`wrangler pages deploy`](
...
+{{</Aside>}}
+
+Manage your Workers [Queues](/queues/) configurations.
+
+### `create`
+
+Create a new Queue.
+
+```txt
+wrangler queue create <name> [OPTIONS]
```
Comment (rts-rob) [inline content/workers/wrangler/commands.md]: Should be `wrangler queues create...` (note the plural)

File: content/workers/wrangler/commands.md [hunk 2]
```diff
@@ -1704,6 +1705,92 @@ This command has been deprecated as of v3 in favor of [`wrangler pages deploy`](
...
+  - The name of the queue to create.
+
+{{</definitions>}}
+
+### `delete`
+
+Delete an existing queue.
+
+```txt
+wrangler hyperdrive delete <name> [OPTIONS]
```
Comment (rts-rob) [inline content/workers/wrangler/commands.md]: Should be `wrangler queues delete...` (note the service name).

File: content/workers/wrangler/commands.md [hunk 3]
```diff
@@ -1704,6 +1705,92 @@ This command has been deprecated as of v3 in favor of [`wrangler pages deploy`](
...
+  - The name of the queue to delete.
+
+{{</definitions>}}
+
+### `list`
+
+List all queues in the current account.
+
+```txt
+wrangler queues list
```
Comment (rts-rob) [inline content/workers/wrangler/commands.md]: Has `[OPTIONS]` for pagination

File: content/workers/wrangler/commands.md [hunk 4]
```diff
@@ -1704,6 +1705,92 @@ This command has been deprecated as of v3 in favor of [`wrangler pages deploy`](
...
+### `consumer`
+
+Manage queue consumer configurations.
+
+### `consumer add <script-name>`
+
+Add a Worker script as a [queue consumer](/queues/learning/how-queues-works/#consumers).
+
+```txt
+wrangler queues consumer add <queue-name> <script-name>
```
Comment (rts-rob) [inline content/workers/wrangler/commands.md]: Has `[OPTIONS]` for e.g., batch size.

**Review by rts-rob — CHANGES_REQUESTED** [review]
Some copy/paste errors and options callouts.

File: content/workers/wrangler/commands.md [hunk 5]
```diff
@@ -29,6 +29,7 @@ Wrangler offers a number of commands to manage your Cloudflare Workers.
 - [`secret:bulk`](#secretbulk) - Manage multiple secret variables for a Worker.
 - [`tail`](#tail) - Start a session to livestream logs from a deployed Worker.
 - [`pages`](#pages) - Configure Cloudflare Pages.
+- [`queues`](#queues) - Configure Workers Queues
```
Comment (Maddy-Cloudflare) [inline content/workers/wrangler/commands.md]:
```suggestion
- [`queues`](#queues) - Configure Workers Queues.
```

File: content/workers/wrangler/commands.md (hunk 1 above)
Comment (Maddy-Cloudflare) [inline content/workers/wrangler/commands.md]:
```suggestion
wrangler queues create <name> [OPTIONS]
```

File: content/workers/wrangler/commands.md (hunk 2 above)
Comment (Maddy-Cloudflare) [inline content/workers/wrangler/commands.md]:
```suggestion
wrangler queues delete <name> [OPTIONS]
```

File: content/workers/wrangler/commands.md (hunk 3 above)
Comment (Maddy-Cloudflare) [inline content/workers/wrangler/commands.md]:
```suggestion
wrangler queues list [OPTIONS]
```

File: content/workers/wrangler/commands.md (hunk 4 above)
Comment (Maddy-Cloudflare) [inline content/workers/wrangler/commands.md]:
```suggestion
wrangler queues consumer add <queue-name> <script-name> [OPTIONS]
```

**Review by Maddy-Cloudflare — APPROVED** [review]

**Review by rts-rob — APPROVED** [review]

**Pull request merged**